drivers/hmis/app/graphql/types/hmis_schema/has_external_form_submissions.rb

```diff
@@ -23,7 +23,7 @@ def external_form_submissions_field(name = :external_form_submissions, descripti
       def resolve_external_form_submissions(scope = object.external_form_submissions, filters: nil)
         return [] unless current_user.can_manage_external_form_submissions?
 
-        scope = scope.apply_filters(filters)
+        scope = scope.apply_filters(filters) if filters.present?
```

Comment:
@gigxz this change means that spam messages will be included by default. Is that what you intended?